src/documents/css/idg.css.styl

```diff
@@ -355,11 +355,44 @@ setActiveCategory(inputColor) {
 
 /* Styles specific to small screens. Using Foundation 5 value for small screen width */
 @media only screen and (max-width: 40em) {
-    .docs-template-articleBody h1 {
-        font-size: 1.7rem;
+
+    * {
+        word-wrap: break-word;
```

Comment:
If we do need this, can we also add hyphens, which would hopefully make things look better at least in the browsers that support it.

https://css-tricks.com/almanac/properties/h/hyphenate/